administrator-manual/en/backup.rst

```diff
@@ -125,14 +71,23 @@ Other restored configurations:
 
 Steps to be executed:
 
-1. Install the new machine with the same host name as the old one
-2. Configure a data backup, so the system can retrieve saved data and configuration
```

Comment:
We are now missing this information:
the configuration backup can be directly restore even from data backup.